RadarSDK/Radar.m

```diff
@@ -1051,6 +1051,10 @@ - (void)applicationWillEnterForeground {
             [[RadarLocationManager sharedInstance] updateTrackingFromMeta:meta];
         }];
     }
+    
+    [[RadarAPIClient sharedInstance] sendEvent:@"app_open" withMetadata:nil user:nil trackEvents:nil completionHandler:^(RadarStatus status, NSDictionary * _Nullable res, NSArray<RadarEvent *> * _Nullable events) {
```

Comment:
is it required to pass `withMetadata`, `user` and `trackEvents` with the value `nil` or can we remove these?

Reply:
i think it's required, objective-c doesnt have default parameters. we could define new function signatures to emulate default params, but that would be a less clean solution